docs: add a note on liquibase execution at GW startup (#445)
gravitee-io/release: upgrades/3.x/3.10.11/README.adoc

- Platform response policies are executed *after* API response policies

== Liquibase execution

> If you use a JDBC repository, we recommend you disabling the liquibase scripts execution by the gateway. It will avoid some conflicts when upgrading and starting Gateway and Management API in the same time.

Reviewer: ```suggestion
If you use a JDBC repository, we recommend you disable the liquibase scripts execution by the gateway. It will avoid some conflicts when upgrading and starting Gateway and Management API at the same time.
```

- Platform response policies are executed *after* API response policies

== Liquibase execution

> From this version, the gateway no longer runs liquibase at startup. It will avoid some conflicts when upgrading and starting Gateway and Management API in the same time.

Reviewer: Some rewording suggestion. WDYT ?
```suggestion
From this version, the gateway no longer runs liquibase database migration at startup. It avoids conflicts when upgrading and starting Gateway and Management API at the same time.
```
